src/Equinor.ProCoSys.Config/CommWebApp/Auth.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Linq;
+using System.Web.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using ConflictResult = Microsoft.AspNetCore.Mvc.ConflictResult;
+
+namespace Equinor.ProCoSys.Config.CommWebApp
+{
+    public static class Auth
+    {
+        [FunctionName("CommWebAppAuth")]
+        public static IActionResult Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "CommWebApp/Auth")] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("Processing configuration request...");
+
+            var configConnectionString = Environment.GetEnvironmentVariable("FrontendConfig");
```

Comment:
This needs to be setup using a new CommWebAppConfig with a new App Configuration instance

Reply:
This is now updated from "FrontendConfig" to "CommWebAppConfig". CommWebAppConfig has been added as a key in pcs-config-non-prod-func > Configuration, with the value being the endpoint to a separate app config instance for the Commissioning webapp.